CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,42 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This looks fairly interesting. I think we should merge this with #1404.

Reply:
Contributor Covenant Code of Conduct seems like a mouthful. Couldn't it be shortened to Contributor Code of Conduct? I always think of pentagrams and black cats when I hear covenant.

Reply:
Agreed. On both.

Reply:
After a lot of thought, I don't think merging the two together would keep the same purpose of each as individual documents. In a business setting, this would be the Code of Conduct, but the other one would be the ethics and culture statements. Both attempt to lead people in the right direction, but have two completely different tones. Rather than trying to weave them together, I think putting the main points of the other document as a subsection about project etiquette would maintain the general feel of both without losing the intention of either.

Reply:
I have to slightly disagree on this. 

In a FOSS project you need both extensive but not tiring documentation. 

A Code of Conduct achieves exactly and - just - that: introducing people to the ethics of a community. In our case, the reicast community. Establishing the proper way of communicating and collaborating means also understanding the ethics and culture of a community. 

Contributor Code of Conduct is describing the doc on point.

Reply:
@skmp recommended merging the two. I am stating that they are two completely different tones and the "new and important" points of the other one may be better suited as a subsection of this, more professionally worded, document over flat out combining them.
You may also want to take into consideration the fact that this document seems more geared at contributors / members, while the other covers a broader range of communications.